cli_tools/common/utils/validation/validation_utils.go

```diff
@@ -44,3 +46,10 @@ func ValidateFqdn(flagValue string, flagKey string) error {
 	}
 	return nil
 }
+
+func ValidateRfc1035Label(value string) error {
+	if len(value) > 63 || !rfc1035LabelRegexp.MatchString(value) {
```

Comment:
I am not sure how is the disk name decided here. Is it provided by user input, or generated by import workflow? If prior, we may explicitly tell them the length exceeds limit.

Reply:
Customer provides instance name, we decide on disk names. Customer can provide an instance name up to 63 characters, but the problem is disk names have the same limitation while possibly there are multiple disks. So we run into the problem of disk names exceeding 63 characters if we just append a suffix to instance name.

Reply:
Is there a hard requirement that disk name has to contain the full instance name? If not, we can just keep a partial instance name + a suffix. This is how daisy workflow handle the similar situation today - for example, when worker instance name + id exceeds 63, the name will be shortened, just kept a partial name + id.

Reply:
That's what generateDataDiskName() does.

Reply:
perfect!